account-kit/react/src/components/auth/card/add-passkey.tsx

```diff
@@ -44,7 +45,10 @@ export const AddPasskey = () => {
   if (authStep.error) {
     return (
       <ConnectionError
-        connectionType="passkey"
+        headerText={ls.error.connection.passkeyTitle}
+        bodyText={ls.error.connection.passkeyBody}
+        icon={<PasskeyConnectionFailed />}
+        shouldDisconnect={false}
         handleTryAgain={addPasskey}
         handleUseAnotherMethod={() => setAuthStep({ type: "complete" })}
```

Comment:
What is the expected behaviour here?  ATM when you click handleUseAnotherMethod the modal closes and nothing happens.  Should we disconnect the user and revert to initial state or prevent this button from rendering in this state?

Reply:
I wonder if we want to show this at all actually. For Add Passkey we should just have a Try Again CTA otherwise folks can just exit out of the flow

Reply:
added conditional render for the handleUseAnotherMethod